common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -335,6 +336,7 @@ public static byte[] uncompress(final byte[] src) throws IOException {
         return result;
     }
 
+    @Deprecated
```

Comment:
It's better to add link to new method, like this

/**
  * @deprecated use {@link Compressor#compress()} instead. 
  */